src/modules/PublicLab.MainImageModule.js

```diff
@@ -21,25 +21,19 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
     _module.image = new Image();
 
     _module.value = function(url, id) {
-
-      if (typeof url == 'string') {
-         
-             _module.image.onload  = function(){
-                  var height_dropdown=this.height;
-                  var width_dropdown=this.width;
-                  if(this.width>700){
-                     var aspect_ratio=this.width/800
-                     width_dropdown=800
-                     height_dropdown=this.height/aspect_ratio   
-                  }
-                _module.dropEl.css('height',height_dropdown)
-                _module.dropEl.css('width',width_dropdown)
-                _module.dropEl.css('background-size',width_dropdown+'px '+height_dropdown+'px');
-
-              }
-            
-        
-          
+        if (typeof url == 'string') {
+            _module.image.onload  = function(){
+            var height_dropdown=this.height;
```

Comment:
This should be formatted like this `var height_dropdown = this.height;`.
You can apply the same to other lines.